en/setup/dev_env_linux.md

```diff
@@ -33,7 +33,7 @@ Then follow the instructions for your development target in the sections below.
 
 To install the development toolchain:
 
-1. Download <a href="https://raw.githubusercontent.com/PX4/Devguide/{{ book.px4_version }}/build_scripts/ubuntu_sim_nuttx.sh" target="_blank" download>ubuntu_sim_nuttx.sh</a>.
+1. Download <a href="https://raw.githubusercontent.com/PX4/Devguide/{{ book.px4_version }}/build_scripts/ubuntu_sim_nuttx.sh" target="_blank" download>ubuntu_sim_nuttx.sh</a>. It's best to use `wget`.
```

Comment:
I would add the actual command:
```
1. Download the script:
    ```bash
    wget https://raw.githubusercontent.com/PX4/Devguide/{{ book.px4_version }}/build_scripts/ubuntu_sim_nuttx.sh
    ```

Reply:
By placing the command in a code block how are we able to use interpolation?
```bash
wget https://raw.githubusercontent.com/PX4/Devguide/{{ book.px4_version }}/build_scripts/ubuntu_sim_nuttx.sh
```

Reply:
I don't know if the gitbook magic can do it @hamishwillee ?

Reply:
I'm not seeing anything like that in their [documentation](https://docs.gitbook.com/content-editing/markdown)

The link points to master, could we just make that static if we can't use interpolation?

Reply:
There is no way to do the interpolation in a code block. The link points to master for this build, but if you look at book.json where it is defined, you will see that on 1.9.0 it points to the 1.9.0 branch. That is why my trees and Firmware match, and why I did it the way that I did rather than wget.

So, for this case, suggest just use "master" with no interpolation because in the very near future I hope to replace all this with scripts in the firmware tree. Then I could just say "sync Firmware via git, then run the file ....".